analytical/templatetags/piwik.py

```diff
@@ -14,6 +14,8 @@
 from analytical.utils import (is_internal_ip, disable_html,
                               get_required_setting, get_identity)
 
+import warnings
+warnings.warn('The Piwik module is deprecated; use the Matomo module.', DeprecationWarning)
```

Comment:
Alright, that covers the suggested warning mentioned above. :heavy_check_mark:

Reply:
The Piwik integration code here is irrelevant. It should simply inherit from Matomo.